plugins/mysql/mysql_disk_by_user

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
```

Comment:
This was probably only relevant for python2.

Reply:
Removed.